fiftyone/core/config.py

```diff
@@ -197,6 +197,12 @@ def __init__(self, d=None):
         self.timezone = self.parse_string(
             d, "timezone", env_var="FIFTYONE_TIMEZONE", default=None
         )
+        self.disable_automatic_dataset_cleanup = self.parse_bool(
```

Comment:
The trouble here is that, in a multiple connection environment, everybody would have to set this to `True` to actually prevent dataset cleanup.

Reply:
Yeah I was thinking about a single user that might happen to have their own instance of Mongo already running, not a team. But I can remove that no problem.